staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -298,7 +299,10 @@ func (s *WatchServer) HandleWS(ws *websocket.Conn) {
 			obj := s.Fixup(event.Object)
 			if err := s.EmbeddedEncoder.Encode(obj, buf); err != nil {
 				// unexpected error
-				utilruntime.HandleError(fmt.Errorf("unable to encode watch object %T: %v", obj, err))
+				err1 := fmt.Errorf("unable to encode watch object %T: %v", obj, err)
+				utilruntime.HandleError(err1)
+				klog.Error(err1)
```

Comment:
Actually now that you call my attention to it, can you put the logging back the way it was? Empirically, it already shows up in the logs if it happens.

Reply:
The info log I added yesterday was prior to calling EmbeddedEncoder.Encode()

I wonder how useful it is - since there is no error at that point
it may also spam the log file

Reply:
No, I mean before you made any changes; there's no need to log this twice. HandleError already logs it.

Reply:
@lavalamp 
I have simplified the patch